doc/man3/EVP_seed.pod

```diff
@@ -0,0 +1,61 @@
+=pod
+
+=head1 NAME
+
+EVP_seed_cbc,
+EVP_seed_cfb,
+EVP_seed_ecb,
+EVP_seed_ofb
+- EVP SEED cipher
+
+=head1 SYNOPSIS
+
+=for comment generic
+
+ #include <openssl/evp.h>
+
+ const EVP_CIPHER *EVP_ciphername(void)
+
+I<EVP_ciphername> is used a placeholder for any of the described cipher
+functions, such as I<EVP_seed_cbc>.
```

Comment:
Please enumerate them here as well, and don't forget to remove `=for comment generic` when you do

Reply:
Done.